docs/how-to-guides/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -155,13 +155,15 @@ add_action( 'init', function() {
 } );
 ```
 
-## Child InnerBlocks: Parent and Ancestors
+## Using Hierarchical Context in InnerBlocks
 
-A common pattern for using InnerBlocks is to create a custom block that will be included only in the InnerBlocks.
+A common pattern for using InnerBlocks is to create a custom block that will be included _only_ in the InnerBlocks. This allows InnerBlocks to have a hierarchical context for nesting child blocks.
```

Comment:
I feel like this is still somewhat convoluted / complex.

Maybe something like this would work better:

```suggestion
A common pattern in the editor are blocks that should only be available to users when they are nested inside specific other blocks. We have two mechanisms for that. A block can define its `parent` when the block needs to be a direct descendant of said block, or it can define an `ancestor` if it can appear anywhere nested within said block.
```